src/blueapi/service/model.py

```diff
@@ -169,6 +170,37 @@ class EnvironmentResponse(BlueapiBaseModel):
     )
 
 
+class SourceInfo(str, Enum):
+    pypi = "pypi"
+    scratch = "scratch"
```

Comment:
Nit: Make the same as other enums in the API

```suggestion
    PYPI = "pypi"
    SCRATCH = "scratch"
```